src/plugins/data_view_editor/public/components/data_view_editor_flyout_content.tsx

```diff
@@ -376,6 +385,17 @@ const IndexPatternEditorFlyoutContentComponent = ({
         <EuiTitle data-test-subj="flyoutTitle">
           <h2>{editData ? editorTitleEditMode : editorTitle}</h2>
         </EuiTitle>
+        {showManagementLink && editData && editData.id && (
```

Comment:
I think we should hide it when you edit an ad-hoc data view (now we show it when you edit ad-hoc data view)

Reply:
Good point, added that